x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/GetConnectorAction.java

```diff
@@ -65,28 +70,28 @@ public ActionRequestValidationException validate() {
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
-            super.writeTo(out);
-            out.writeString(connectorId);
+            TransportAction.localOnly();
```

Comment:
I don't know if this impacts us but this will be removed at some point. Probably fine though?

https://github.com/elastic/elasticsearch/blob/412214435564f508a6b6c61866f3a36ee2cefe24/server/src/main/java/org/elasticsearch/action/support/TransportAction.java#L177-L186

Reply:
Yup it should be fine, Jim suggested to follow this pattern.